AdvantageBenchmark/privateBuild/Program.cs

```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
             if (args.Length > 1) tstMax  = Convert.ToInt32(args[1]);
             if (args.Length > 2) loopCnt = Convert.ToInt32(args[2]);
 
-            using (var sim = new QuantumSimulator()) 
+            using (var sim = new QuantumSimulator(typeof(Microsoft.Quantum.Intrinsic.TargetIntrinsics))) 
```

Comment:
Not of all simulators, as some simulators may decide to provide their own overrides instead of needing to find the interface shims. I have some ideas about how to remove the need for this, which I'll expound on in the other comments.